I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/extremis/coteries/orks/evts_RejOrks.rpy", line 71, in script
    $ SetValCarac(trait.Serenite.NOM, 0)
  File "game/extremis/coteries/orks/evts_RejOrks.rpy", line 71, in <module>
    $ SetValCarac(trait.Serenite.NOM, 0)
NameError: name 'SetValCarac' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/extremis/coteries/orks/evts_RejOrks.rpy", line 71, in script
    $ SetValCarac(trait.Serenite.NOM, 0)
  File "renpy/ast.py", line 922, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 2218, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/extremis/coteries/orks/evts_RejOrks.rpy", line 71, in <module>
    $ SetValCarac(trait.Serenite.NOM, 0)
NameError: name 'SetValCarac' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.4.1439
DestinExtremis 1.0
Thu Jun  3 14:37:35 2021
